module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -3415,6 +3467,14 @@ protected function getItemsWithBibsForTransactions(
                 $bibIdsToItems[$this->extractId($bibId)][$itemId] = true;
             }
         }
+        if ($this->config['InnReach']['enabled']) {
```

Comment:
Probably worth either adding the same `?? false` here as above, or creating an `innReachEnabled()` method to encapsulate the logic if you prefer.